room/compiler-processing/src/main/java/androidx/room/compiler/processing/XAnnotated.kt

```diff
@@ -52,6 +63,7 @@ interface XAnnotated {
      * Returns `true` if this element has an annotation that is declared in the given package.
      */
     // a very sad method but helps avoid abstraction annotation
+    // Possibly deprecate in favor of `getAllAnnotations`?
```

Comment:
this was a hacky thing i added because we need this in 1 place in room for auto-value and i didn't want us to parse all annotations just to get that.

Reply:
Makes sense - do you want me to remove this comment or do you imagine there could be a follow up to add a cleaner API for this purpose?

Reply:
lets just remove the comment. Can still link to the other methods.